com.reprezen.swagedit.openapi3/resources/templates.xml

```diff
@@ -0,0 +1,328 @@
+       enabled="true">patch:
+      summary: Partial update
+      description: ${Update}
+      parameters:
+        - name: ${request}
+          in: body
+          required: true
+          schema:
+           $$ref: ${model}  
+      responses:
+        '${200}':
+          description: OK
+          schema:
+            $$ref: ${model}</template>
+
+    <template name="options" 
+       id="com.reprezen.swagedit.openapi3.templates.path_options_template" 
+       description="OPTIONS template" 
+       context="com.reprezen.swagedit.openapi3.templates.path_item" 
+       enabled="true">options:
+      summary: Request resource options
+      description: Request information about methods available for resource manipulation 
+      responses:
+        '${200}':
+          description: OK</template>
+
+    <template name="head" 
+       id="com.reprezen.swagedit.openapi3.templates.path_head_template" 
+       description="HEAD template" 
+       context="com.reprezen.swagedit.openapi3.templates.path_item" 
+       enabled="true">head:
+      summary: Query a resource in the server
+      description: HEAD sends request and get the response only in the header, the body should not be used
+      responses:
+        '${200}':
+          description: OK</template>
+          
+<!--
+	RESPONSES
+ -->
+ 
+	<template name="OK response" 
+       id="com.reprezen.swagedit.openapi3.templates.200_response" 
+       description="OK template" 
+       context="com.reprezen.swagedit.openapi3.templates.responses" 
+       enabled="true">2${00}:
+      description: ${OK}</template>
+
+    <template name="client error response" 
+       id="com.reprezen.swagedit.openapi3.templates.400_response" 
+       description="client error response template" 
+       context="com.reprezen.swagedit.openapi3.templates.responses" 
+       enabled="true">4${00}:
+      description: ${Client} error</template>
+
+    <template name="server error response" 
+       id="com.reprezen.swagedit.openapi3.templates.500_response" 
+       description="server error response template" 
+       context="com.reprezen.swagedit.openapi3.templates.responses" 
+       enabled="true">5${00}:
+      description: ${Server} error</template>
+
+<!--
+	LINKS 
+ -->
+
+ 	<template name="response link" 
+       id="com.reprezen.swagedit.openapi3.templates.response_link" 
+       description="server response link template" 
+       context="com.reprezen.swagedit.openapi3.templates.response" 
+       enabled="true">links:
+            ${name}:
+              $$ref: ${ref}</template>
+
+<!--
+	PARAMETERS 
+ -->
+ 
+ <template name="path parameter" 
+       id="com.reprezen.swagedit.openapi3.templates.path_parameter_object" 
+       description="path parameter template" 
+       context="com.reprezen.swagedit.openapi3.templates.parameter_object" 
+       enabled="true">name: ${name}
+in: path
+description: ${ID}
+required: true
+type: ${integer}
```

Comment:
parameter type is now defined inside a [schema property](https://github.com/OAI/OpenAPI-Specification/blob/OpenAPI.next/versions/3.0.md#fixed-fields-10):
```yaml
        - name: name
          in: path
          description: ID
          required: true
          schema:
            type: integer
            format: int64
```